README.md

```diff
@@ -23,24 +111,16 @@ Once it's finished processing the default images (< 1 minute) you can then acces
 
 The Image Caption Generator endpoint must be available at `http://localhost:5000` for the web app to successfully start.
 
-### Configuring ports
+### Configuring ports (Optional)
 
 If you want to use a different port or are running the ML endpoint at a different location
 you can change them with command-line options:
 
     python app.py --port=[new port] --ml-endpoint=[endpoint url including protocol and port]
 
-## Instructions for Docker
-
-To run the web app with Docker the containers running the webserver and the REST
-endpoint need to share the same network stack. This is done in the following steps.
-
-Modify the command that runs the Image Caption Generator REST endpoint
-to map an additional port in the container to a port on the host machine.
-In the example below it is mapped to port 8088 on the host but other ports
-can also be used.
+### Instructions for Docker (Optional)
 
-    docker run -it -p 5000:5000 -p 8088:8088 --name max-im2txt max-im2txt
+Alternatively we can use the docker image provided that contains all the dependencies
```

Comment:
this is shaping up nicely, it would be nice to have a screenshot of the running app somewhere in this section.